### PR TITLE
Makes dionae insulated against shocks

### PR DESCRIPTION
**Pull request by Yonnimer**
What it says on the tin. Right now dionae have no way to insulate themselves given they can't wear gloves, this should fix that and make engineering dionae more doable.

### CODE REVIEW

**Comment by Yonnimer** [comment]
!review

File: code/modules/mob/living/carbon/human/species/station/diona/diona.dm [hunk 1]
```diff
@@ -29,7 +29,7 @@
 	slowdown = 7
 	rarity_value = 4
 	hud_type = /datum/hud_data/diona
-	siemens_coefficient = 0.3
+	siemens_coefficient = 0
```
Comment (Alberyk) [inline code/modules/mob/living/carbon/human/species/station/diona/diona.dm:32]: Take into consideration that this also makes them immune to taser and stun beams.

Comment (Yonnimer) [reply code/modules/mob/living/carbon/human/species/station/diona/diona.dm:32]: wasn't aware of this, will work on changing this

Comment (Yonnimer) [reply code/modules/mob/living/carbon/human/species/station/diona/diona.dm]: Should be fixed now

**Review by Chada-1 — APPROVED** [review]

File: code/modules/power/power.dm [hunk 2]
```diff
@@ -355,7 +355,7 @@
 	if(H)
 		if(H.species.siemens_coefficient == 0)
 			return
-		if(isvaurca(H)) //natural vaurca insulation
+		if(isvaurca(H) || H.is_diona()) //natural vaurca and dionae insulation
```
Comment (NonQueueingMatt) [inline code/modules/power/power.dm:358]: Nooooo. If you're doing this, make this a species proc such as:

`/datum/species/proc/is_naturally_insulated()` which returns FALSE by default and returns TRUE on vaurca/diona.

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

File: code/modules/power/power.dm (hunk 2 above)
Comment (Yonnimer) [reply code/modules/power/power.dm]: Changed, works in testing

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**